osu.Game/Screens/Import/FileImportScreen.cs

```diff
@@ -0,0 +1,320 @@
+                                            Children = new Drawable[]
+                                            {
+                                                new Box
+                                                {
+                                                    Colour = overlayColourProvider.Background3,
+                                                    RelativeSizeAxes = Axes.Both
+                                                },
+                                                fileNameScroll = new OsuScrollContainer
+                                                {
+                                                    Masking = false,
+                                                    RelativeSizeAxes = Axes.Both,
+                                                    Child = currentFileText = new TextFlowContainer(t => t.Font = OsuFont.Default.With(size: 30))
+                                                    {
+                                                        AutoSizeAxes = Axes.Y,
+                                                        RelativeSizeAxes = Axes.X,
+                                                        Anchor = Anchor.Centre,
+                                                        Origin = Anchor.Centre,
+                                                        TextAnchor = Anchor.Centre
+                                                    },
+                                                },
+                                            }
+                                        },
+                                    },
+                                    new Drawable[]
+                                    {
+                                        new Container
+                                        {
+                                            RelativeSizeAxes = Axes.X,
+                                            AutoSizeAxes = Axes.Y,
+                                            Children = new Drawable[]
+                                            {
+                                                new Box
+                                                {
+                                                    Colour = overlayColourProvider.Background4,
+                                                    RelativeSizeAxes = Axes.Both
+                                                },
+                                                new FillFlowContainer
+                                                {
+                                                    RelativeSizeAxes = Axes.X,
+                                                    AutoSizeAxes = Axes.Y,
+                                                    Spacing = new Vector2(10),
+                                                    Children = new Drawable[]
+                                                    {
+                                                        new SettingsEnumDropdown<FileFilterType>
```

Comment:
Rather than having this, why not just always show every filetype? I don't see much benefit to being able to filter it.